{
  "title": "PDS Validation Tool Report",
  "configuration": {
    "version": "1.16.0-SNAPSHOT",
    "date": "2019-07-26T15:35:15Z"
  },
  "parameters": {
    "targets": "[file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/PDS-543/ladee_mission_bundle_SIP_4col_manifest_v1.0.xml]",
    "severityLevel": "WARNING",
    "recurseDirectories": "true",
    "fileFiltersUsed": "[*.xml, *.XML]",
    "dataContentValidation": "on",
    "maxErrors": "100000",
    "registeredContextsFile": "/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/main/resources/util/registered_context_products.json"
  },
  "productLevelValidationResults": [
    {
      "status": "PASS",
      "label": "gov.nasa.pds.validate.ValidateLauncher",
      "messages": [
        {
          "severity": "INFO",
          "type": "info.validation.general",
          "message": "number of registered context products used for validation: 1875"
        },
        {
          "severity": "INFO",
          "type": "info.validation.general",
          "message": "Total number of Product Context: 1875"
        }
      ],
      "fragments": [],
      "dataContents": []
    },
    {
      "status": "FAIL",
      "label": "file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/PDS-543/ladee_mission_bundle_SIP_4col_manifest_v1.0.xml",
      "messages": [],
      "fragments": [],
      "dataContents": [
        {
          "dataFile": "file:/Users/jpadams/Documents/proj/pds/pdsen/workspace/validate/src/test/resources/PDS-543/ladee_mission_bundle_SIP_4col_manifest_v1.0.tab",
          "messages": [
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 1,
              "message": "Record length exceeds the max defined record length (expected 59, got 156)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 2,
              "message": "Record length exceeds the max defined record length (expected 59, got 185)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 3,
              "message": "Record length exceeds the max defined record length (expected 59, got 195)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 4,
              "message": "Record length exceeds the max defined record length (expected 59, got 188)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 5,
              "message": "Record length exceeds the max defined record length (expected 59, got 198)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 6,
              "message": "Record length exceeds the max defined record length (expected 59, got 184)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 7,
              "message": "Record length exceeds the max defined record length (expected 59, got 184)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 8,
              "message": "Record length exceeds the max defined record length (expected 59, got 190)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 9,
              "message": "Record length exceeds the max defined record length (expected 59, got 190)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 10,
              "message": "Record length exceeds the max defined record length (expected 59, got 194)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 11,
              "message": "Record length exceeds the max defined record length (expected 59, got 204)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 12,
              "message": "Record length exceeds the max defined record length (expected 59, got 175)."
            },
            {
              "severity": "ERROR",
              "type": "error.table.record_length_mismatch",
              "table": 1,
              "record": 13,
              "message": "Record length exceeds the max defined record length (expected 59, got 175)."
            }
          ]
        }
      ]
    }
  ],
  "summary": {
    "totalErrors": 13,
    "totalWarnings": 0,
    "messageTypes": [
      {
        "messageType": "error.table.record_length_mismatch",
        "total": 13
      }
    ]
  }
}